be/src/formats/orc/orc_file_writer.cpp

```diff
@@ -521,32 +441,10 @@ StatusOr<std::unique_ptr<orc::Type>> ORCFileWriter::_make_schema_node(const Type
     case TYPE_DATE: {
         return orc::createPrimitiveType(orc::TypeKind::DATE);
     }
-    case TYPE_DATETIME:
-        [[fallthrough]];
-    case TYPE_TIME: {
```

Comment:
another tiny bug is that `TYPE_TIME` is not supported actually, so remove it when `make_schema`